vue@latest installs previous version of vue 2.6.14

shadhin-msa opened the pull request:
$ yarn add vue@latest @quasar/extras@latest  downloads previous version of Vue.js . it downloads vue@2.6.14 and vue-router@3.5.3 resulting error.
Error comes up after refreshing node-modules from previous version.
vue & vue-router uses @next tag for their newest release.
their @latest tags are referred to previous version of vue.

reference :
 https://www.npmjs.com/package/vue  > versions
https://www.npmjs.com/package/vue-router  > versions

<!--
  Please make sure to read the Pull Request Guidelines:
  https://github.com/quasarframework/quasar/blob/dev/CONTRIBUTING.md#pull-request-guidelines
-->

<!-- PULL REQUEST TEMPLATE -->
<!-- Update "[ ]" to "[x]" to check a box (space sensitive) -->

**What kind of change does this PR introduce?** <!-- Check at least one -->

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** <!-- Check one -->

- [ ] Yes
- [x] No

<!-- If yes, please describe the impact and migration path for existing applications: -->

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch (or `v[X]` branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) <!-- for faster update click on "Suggest an edit on GitHub" at bottom of page --> or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to [start a new feature discussion](https://github.com/quasarframework/quasar/discussions/new?category=ideas-proposals) first and wait for approval before working on it)

**Other information:**

shadhin-msa committed:
vue@latest installs previous version of vue 2.6.14

$ yarn add vue@latest @quasar/extras@latest  downloads previous version of Vue.js . it downloads vue@2.6.14 and vue-router@3.5.3 resulting error.
Error comes up after refreshing node-modules from previous version.
vue & vue-router uses @next tag for their newest release.
their @latest tags are referred to previous version of vue.

reference :
 https://www.npmjs.com/package/vue  > versions
https://www.npmjs.com/package/vue-router  > versions

MilosPaunovic reviewed (commented):
I think you should change it in line 156, as well.

shadhin-msa committed:
line 156 also needed to change 

line 156 also needed to change  vue@latest to vue@next  vue-router@latest to vue-router@next

shadhin-msa commented:
> I think you should change it in line 156, as well.

thanks updated

rstoenescu committed:
Update upgrade-guide.md

rstoenescu commented:
Thanks for contributing!
Changed it to explicit versions for vue and vue-router, since the "next" tag will soon not be used anymore.
